testing/upload_validation.ts

```diff
@@ -336,6 +336,7 @@ const defaultAuthenticationValidators: Record<AuthenticationType, z.ZodTypeAny>
     authorizationUrl: z.string(),
     tokenUrl: z.string(),
     scopes: z.array(z.string()).optional(),
+    scopeDelimiter: z.string().optional(),
```

Comment:
I forgot why I didn't merge https://github.com/coda/packs-sdk/pull/1273 but one comment from Chris was to limit this to a subset of single characters.

Reply:
+1 - worry about parsing this down the line and missing text.  Let's require this to be one of a known set of single character separators.

Reply:
I imagine that space and comma would give us 99% coverage.

Reply:
Thanks for the suggestion, sending a followup to restrict to specific delimiters only.